.github/workflows/publish-cdn.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish to CDN
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release tag'
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
```

Comment:
Sorry, I should have noticed this in my first-pass review but I don't think you need any elevated permissions for this workflow. You're not pushing to GitHub Packages, which is what `packages` is needed for.

Reply:
Good spot, removed it.